plugins/scoreboard/derma/cl_scoreboard.lua

```diff
@@ -94,6 +94,12 @@ local PANEL = {}
 				elseif (!visible and amount > 0) then
 					v:SetVisible(true)
 				end
+				
+				local faction = nut.faction.indices[k]
+				
+				if (amount != 0) then
+					v:SetVisible(hook.Run("HideFactionInScoreboard", faction) ~= false or LocalPlayer():IsAdmin())
```

Comment:
```suggestion
					v:SetVisible(hook.Run("ShowFactionInScoreboard", faction) ~= false or LocalPlayer():IsAdmin())
```
To make the purpose and syntax of the hook clearer

Reply:
As per Seamus' suggestion